sktime/alignment/dtw_python.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+"""Interface module to dtw-python package.
+
+Exposes basic interface, excluding multivariate case.
+"""
+
+__author__ = ["fkiraly"]
+
+import pandas as pd
+import numpy as np
+
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+from sktime.alignment._base import BaseAligner
+
+_check_soft_dependencies("dtw")
+
+
+class AlignerDTW(BaseAligner):
+    """Aligner interface for dtw-python.
+
+    Behaviour: computes the full alignment between X[0] and X[1]
+        assumes pairwise alignment (only two series) and univariate
+        if multivariate sequences are passed:
+            alignment is computed on univariate series with variable_to_align;
+            if this is not set, defaults to the first variable of X is used
```

Comment:
```suggestion
            if this is not set, defaults to the first variable of X[0]
```
* fixing grammar by removing `is used`
* I think we are aligning first variable in X[0] rather than first variable of X.